ui/src/components/Publication/Creation/KeyInformation.tsx

```diff
@@ -107,21 +124,54 @@ const KeyInformation: React.FC = (): React.ReactElement => {
                     </div>
                     <div className="mt-12">
                         <div className="mb-10">
-                            <Components.PublicationCreationStepTitle text="Affiliated Organisations" />
+                            <Components.PublicationCreationStepTitle
+                                text="Affiliated Organisations"
+                                id="affiliations-header"
+                                required
+                            />
                             <span className="mb-2 block text-sm leading-snug text-grey-700 transition-colors duration-500 dark:text-white-100">
-                                Please enter the details for any organisations this publication is associated with.
-                                Please note that funding sources are added later in the publication form.
+                                Does this publication have any affiliations?
                             </span>
-                            <Components.Button
-                                title="Search for your organisation's ROR"
-                                href="https://ror.org/"
-                                openNew
-                                endIcon={
-                                    <OutlineIcons.SearchIcon className="h-6 w-6 text-teal-500 transition-colors duration-500 dark:text-white-50" />
-                                }
+                            <Components.RadioConfirm
```

Comment:
There's an issue with the props provided to this component
<img width="664" alt="image" src="https://user-images.githubusercontent.com/48569671/226675029-b24c1219-e0dc-475e-8c67-6c215cd2b878.png">

I see 'status' prop is undefined in some cases. That prop should always be a boolean => undefined means false

Reply:
UI state is not updating correctly.
Steps to replicate the issue:
1. Select 'Yes' and add an affiliation
<img width="778" alt="image" src="https://user-images.githubusercontent.com/48569671/226684467-7221379c-034f-40be-826b-b61291b34d13.png">
2. Select 'No', click confirm checkbox and save the publication
3. Select 'Yes' again. The affiliation is deleted from DB but it still shows in the UI because the state was not updated
<img width="848" alt="image" src="https://user-images.githubusercontent.com/48569671/226687651-2e525ee0-b5be-46e7-94a8-edd21e1b3edf.png">
4. Save the publication (DELETE affiliations endpoint is called again, not sure why?!)
5. Refresh the page - affiliations won't show up anymore